src/service/activity-impl.js

```diff
@@ -204,11 +209,12 @@ export class Activity {
   /** @private */
   setUpActivityListeners_() {
     for (let i = 0; i < ACTIVE_EVENT_TYPES.length; i++) {
-      this.unlistenFuncs_.push(listen(this.win_.document.documentElement,
+      this.unlistenFuncs_.push(listen(this.win_.document,
         ACTIVE_EVENT_TYPES[i], this.boundHandleActivity_));
     }
 
     this.viewer_.onVisibilityChanged(this.boundHandleVisibilityChange_);
+    this.viewport_.onScroll(this.boundHandleActivity_);
```

Comment:
Hello  Team,
Please remove my input into "amphtml"
Thank you
Wojtek 

    On Tuesday, February 23, 2016 2:53 PM, Justin Ridgewell <notifications@github.com> wrote:
 

 In src/service/activity-impl.js:>          ACTIVE_EVENT_TYPES[i], this.boundHandleActivity_));
>      }
>  
>      this.viewer_.onVisibilityChanged(this.boundHandleVisibilityChange_);
> +    this.viewport_.onScroll(this.boundHandleActivity_);

viewport.onScroll doesn't return an unlisten function
That's a bug, want to fix it to?  —
Reply to this email directly or view it on GitHub.